docs/runtime-upgrade.md

```diff
@@ -0,0 +1,58 @@
+# Testing a runtime upgrade in local
+This must be done before each new release we do in any of out runtimes.
+Supposing we are testing centrifuge runtime in local:
+
+1. Run the relay chain:
+    ```sh
+    ./scripts/init.sh start-relay-chain
+    ```
+    Open **Rococo Local Testnet** client [here](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer)
+
+1. Run the parachain using the last available release:
+    ```sh
+    PARA_CHAIN_SPEC=centrifuge-local CC_DOCKER_TAG=<DOCKER_TAG> ./scripts/init.sh start-parachain-docker
+    ```
+    The `DOCKER_TAG` has the following format: `test-parachain-<DATE>-<COMMIT_HASH>`.
+    You can see the available `DOCKER_TAG`s [here](https://hub.docker.com/r/centrifugeio/centrifuge-chain/tags)
+
+    Open **Centrifuge Local** client [here](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2Flocalhost%3A9946#/explorer)
```

Comment:
This port should not work since the port is derived on the specific para-id relative to the runtime:

> --ws-port $((9946 + $para_id))

Reply:
mmm... It is working with that port. Could it be because what I'm running is `start-parachain-docker` instead of `start-parachain` where it is working as you say?

Reply:
ah, yes!